### PR TITLE
Add Billing post

### PR DESCRIPTION
**Pull request by ivanaur**

### CODE REVIEW

File: billing_5c06ad9604286304a71cfa5b.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Your current month's running total is always reduced by $20 you get each month for your first organization. If you have any discounts, your running total will be reduced by that, too.
+
+## Upgrading to a paid plan
+
+In order to upgrade to a paid plan, go to the Billing page and click on **Upgrade Plan...** button in the top right corner. After that, enter your credit card information and that’s it.
+
+## Invoices
+
+At the bottom of the Billing page, you can find and download your organization’s invoices. You will also receive an invoice every month to the email address you entered while adding your credit card.
```
[the remark this answers is not in the document]
Comment (ivanaur) [reply billing_5c06ad9604286304a71cfa5b.md]: We don't have invoices on the Billing page or we don't send invoices to users?

**Comment by fbrdaric** [comment]
👍

**Comment by ivanaur** [comment]
@fbrdaric @radwo @milica-nerlovic Changed the doc based on your suggestions. Please take a look.

**Comment by milica-nerlovic** [comment]
:+1:

**Pull request merged**